src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -19,9 +19,15 @@
 class SaaSQueryConfig(QueryConfig[SaaSRequestParams]):
     """Query config that generates populated SaaS requests for a given collection"""
 
-    def __init__(self, node: TraversalNode, endpoints: Dict[str, Endpoint]):
+    def __init__(
+        self,
+        node: TraversalNode,
+        endpoints: Dict[str, Endpoint],
+        connector_params: Dict[str, Any],
```

Comment:
Whoever is looking at this class may not know `connector_params` could contain secrets. It's best we're explicit about this. For instance, this is set to the value of `self.secrets` by the `SaaSConnector` class. Otherwise run the risk of having these secrets accidentally returned by the API should anyone not realise.

Reply:
Good callout, I changed the variable name from `connector_params` to `secrets`.